ethsigner/signer/multikey/src/main/java/tech/pegasys/ethsigner/signer/multikey/ThrowingTomlTable.java

```diff
@@ -33,4 +33,12 @@ public Long getLong(final String key) {
     return Optional.ofNullable(table.getLong(key))
         .orElseThrow(() -> new IllegalArgumentException(key + " was not specified in TOML input."));
   }
+
+  public Optional<Boolean> getOptionalBoolean(final String key) {
```

Comment:
nit: Not sure where this should go - maybe here ... just worried that this was deliberately a 'Throwing' TomlTable ... and this doesn't throw... having said that - haven't looked to see how this would fit (maybe the Boolean isn't optional, maybe we force people to specify their TLS on/off, does it need to be optional?)

Reply:
From usability perspective, this keep it backward compatible while making TLS enabled by default and assuming user doesn't require to set trust store by default as well. Having said that, I don't mind forcing user to specify TLS on/off. Though we can't force to specify trust information, that MUST remain optional.

Reply:
I can always bring it to a different file or access it directly as you are correct, this was a deliberate "ThrowableTomlTable"

Reply:
Introduced abstraction. removed the optional methods from this file.